.github/workflows/ios-app-nightly.yml

```
@@ -0,0 +1,64 @@
+          fi
+
+  build-ios-app:
+    timeout-minutes: 15
+    needs: check-changes
+    if: needs.check-changes.outputs.changes-detected == 'true' || github.event_name == 'workflow_dispatch'
+    name: 🍎 Build iOS App
+    runs-on: ubuntu-24.04
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: ./.github/actions/setup-jdk-gradle
```

actionlint (1.7.7)

description is required in metadata of "Setup JDK & Gradle" action at "/home/jailuser/git/.github/actions/setup-jdk-gradle/action.yml"

(action)